docs/content/usage/configuration/credentials.md

```diff
@@ -5,6 +5,27 @@ draft: false
 description: "Configure credentials that the Infection Monkey will use for propagation."
 ---
 
-On this screen you can feed the Infection Monkey "stolen" credentials from your network, simulating an attacker with inside knowledge.
+On this screen, you can configure credentials for Infection Monkey to use during propagation.
+These can be anything from default or weak passwords, to "stolen" credentials from your
+network, simulating an attacker with inside knowledge.
```

Comment:
I think this section can be deleted

Reply:
I think the content reads better with this line.